static/js/place/place.ts

```diff
@@ -109,36 +114,31 @@ function adjustMenuPosition(): void {
 /**
  * Get the geo json info for choropleth charts.
  */
-async function getGeoJsonData(
+export async function getGeoJsonData(
```

Comment:
for cleanup, probably want to put these exported functions in a utils/fetcher file .. since they're not used by place page initial load

Reply:
Good idea. Created fetch.ts to hold them